editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -612,12 +613,11 @@ export function handleKeyDown(
         return [EditorActions.toggleHidden()]
       },
       [INSERT_IMAGE_SHORTCUT]: () => {
-        if (isViewer) {
-          return []
-        }
-        if (isSelectMode(editor.mode) || isInsertMode(editor.mode)) {
-          // FIXME: Side effects.
-          insertImage(dispatch)
+        if (allowedToEdit) {
+          if (isSelectMode(editor.mode) || isInsertMode(editor.mode)) {
+            // FIXME: Side effects.
```

Comment:
I think this FIXME can just go in the bin because at this point the intention is pretty much lost

Reply:
Changed it to indicate that there is a side effect, removing the "FIXME" in c2e2e1ea4.